src/goto-instrument/accelerate/scratch_program.cpp

```diff
@@ -39,7 +39,8 @@ bool scratch_programt::check_sat(bool do_slice)
   symex.constant_propagation=constant_propagation;
   goto_symex_statet::propagationt::valuest constants;
 
-  symex(symex_state, functions, *this);
+  symex.symex_with_state(symex_state, functions, *this, new_symbol_table);
+  ns=namespacet(symbol_table, new_symbol_table);
```

Comment:
Suggest this should be initialised in `scratch_programt`'s constructor, since both `symbol_table` and `new_symbol_table` are (presumably `new_symbol_table` is harmlessly empty until this has happened in any case?)